### PR TITLE
feat(DiffFlameGraph): Add "how to" infos

### PR DESCRIPTION
**Pull request by grafakus**
### ✨ Description

**Related issue(s):** relates to https://github.com/grafana/explore-profiles/issues/224

This PR adds a collapsible section when landing on the "Diff flame graph" view and the user has not selected both flame graphs time ranges. The collapsed section includes an animated gif:

| Before | After |
|  ---   |  ---  |
| image | Screenshot 2024-10-18 at 11 34 11 |
|  |Screenshot 2024-10-18 at 11 34 47|

### 📖 Summary of the changes

The E2E screenshots have also been updated. See diff tab for specific comments.

### 🧪 How to test?

`-`

### CODE REVIEW

File: webpack.config.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ const config = async (env): Promise<Configuration> => {
       extensions: ['.ts', '.tsx', '.js', '.json'],
       alias: {
         '@shared': path.resolve(__dirname, './src/shared'),
+        '@img': path.resolve(__dirname, './src/img'),
```
Comment (grafakus) [inline webpack.config.ts:49]: Convenience

**Comment by Rperry2174** [comment]
let's make the "how" uncollapsed by default -- one less click for the user

**Comment by bryanhuhta** [comment]
> 1. let's make the "how" uncollapsed by default -- one less click for the user. without it being visibile by default we run the same issue of someone potentially missing it and not knowing what to do

Just my 2¢, but this is a double-edged sword. Having the section expanded by default will likely help newcomers understand how to use the diff view on their first impression. However beyond that, I expect an auto-expanded "how-to" like this would be extraordinarily annoying and disruptive beyond the first one or two page lands. Once a user understands the flow, prominently displaying a large gif on how to use the tool which you must scroll by our manually collapse will not be a great experience. This is definitely a trade-off between "good first experience" and "good general experience." Do we want to design for the new user or for the general user? I would recommend against an auto-expanded help, but I will not die on this hill.

**Comment by grafakus** [comment]
I agree with Bryan, it's one more click for the user, _once_.

Let's try to make baby steps and divide & conquer. We can change and iterate later on.

**Pull request merged**